### ISSUE TITLE
API returns 400 error for cell types containing an underscore (_)

### ISSUE THREAD

**Issue by YingX97** [fabilab/cell_atlas_approximations_API#35]
Hi Fabio, 
I try to query the API with cell types that contain an underscore (_), and it returns a 400 error.

`
> let api = require('@fabilab/atlasapprox')
> let params = {organism:"a_thaliana",organ:"shoot",celltype:"proliferating_epi"}
> await api.markers(params)
Uncaught:
{
  status: 400,
  message: 'Cell type not found: proliferating epi.',
  error: {
    type: 'invalid_parameter',
    invalid_parameter: 'celltype',
    invalid_value: 'proliferating epi'
  }
}`

cheers,
Ying

**Comment by iosonofabio** [comment]
yep, it's a bug generated by an attempt to be nice to the chatbot. I'm "fixing' it now as in no cell types will have underscores, but the user can still use underscores and they will be converted into spaces.

Better ideas welcome
